tests/postman/DPS-Service-Api.postman_collection.json

```diff
@@ -760,4 +755,4 @@
 		}
 	],
 	"protocolProfileBehavior": {}
-}
+}
```

Comment:
newline

Reply:
the postman export does not add a line, I have to open the file and add the newline manually, should we?

Reply:
whoopsie